dashboard/db/migrate/20190515203100_add_metrics_to_user_feedbacks.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsToUserFeedbacks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :teacher_feedbacks, :student_visit_count, :integer
+    add_column :teacher_feedbacks, :student_initial_visited_at, :timestamp
+    add_column :teacher_feedbacks, :student_most_recent_visited_at, :timestamp
```

Comment:
sorry to nit, but what do you think about calling these `student_first_visited_at` and `student_last_visited_at`? `[first|last]_xxx_at` follow a pattern used elsewhere in our codebase and read more clearly (in my opinion). I know these are kind of annoying to change in db migrations, but I think we'll be happy later for these to have great names!

Reply:
that's quite alright, sounds good to me!

Reply:
If you don't like those, I think `student_initially_visited_at` and `student_most_recently_visited_at` would read a bit more easily and would be closer to what you originally had.

Reply:
i think following the first/last convention makes sense. naming is hard 😆